db/migrations/20201214175739_jobs_table.sql

```diff
@@ -0,0 +1,356 @@
+-- migrate:up
+create table jobs (
+  id binary(16) not null default (uuid_to_bin(uuid(), 1)),
+  user_id binary(16) not null,
+  system_id binary(16) not null,
+  definition JSON not null,
+  status enum('created', 'queued', 'complete', 'error') default 'created',
+  created_at timestamp not null default current_timestamp,
+  modified_at timestamp not null default current_timestamp on update current_timestamp,
+  primary key (id),
+  key jobs_user_id_key (user_id),
+  foreign key (user_id)
+  references users(id)
+  on delete cascade on update restrict
+) engine=innodb row_format=compressed;
```

Comment:
Are we still working under the idea that jobs would have a set lifetime? Maybe we need an expiration column here.

Reply:
Yeah I'm not sure exactly how that will work, but not a bad idea. I still think I'd rather add the column later if necessary.